code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -459,28 +426,26 @@
 		sinPerson.reagents.add_reagent("nutriment",1000)
 
 /proc/sin_Pride(var/mob/living/carbon/human/sinPerson, var/isIndulged)

```

Comment:
I would just remove it if I could. Sadly, reverting is hard. "Improve not remove" is worst meme ever.